### ISSUE TITLE
Breaks Switcher function in recent version of IntelliJ

### ISSUE THREAD

**Issue by honeySleepr** [Olof-IL/RecentTabOrder#3]
Hi! Your plugin is great ;)
There's an issue when used with the latest version of IntelliJ(2024.3.1 (Ultimate Edition)).

The tab ordering works fine, but the switcher function(Ctrl+Tab) doesn't work like it did before.
Ctrl+Tab should switch to the previously opened file, but with this plugin enabled the files open in a certain rotation.(Disabling the plugin fixes the issue)

**Comment by Olof-IL** [comment]
Thank you for the praise, and the report, I'll look into it as soon as I can!

**Comment by Olof-IL** [comment]
Unfortunately this seems hard to fix in the current version. They've changed how they keep the history for the switcher, and all relevant members are private...

I'll see if I can make some workaround, but don't expect a quick fix.
